dev-libs/libxlsxwriter/libxlsxwriter-1.1.4.ebuild

```diff
@@ -17,6 +17,7 @@ IUSE="openssl"
 
 DEPEND="
 	sys-libs/zlib[minizip]
+	openssl? ( dev-libs/openssl )
```

Comment:
:=

Reply:
Where should I check to figure out whether I need the slot operator or not?

Reply:
If we:
1. link against a package;
2. the package exposes a subslot;
3. the subslot its for its SONAME (unless stated otherwise in metadata.xml, it usually is)
we should use :=.

Reply:
Thank you so much! I think I need to read up on subslots now!